src/test/kotlin/step2/StringCalculatorKoTests.kt

```diff
@@ -0,0 +1,91 @@
+package step2
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.startWith
+import java.lang.IllegalArgumentException
+
+class StringCalculatorKoTests : FreeSpec({
+    lateinit var stringCalculator: StringCalculator
+
+    beforeAny {
+        stringCalculator = StringCalculator()
+    }
+
+    "정상 작동" - {
+        listOf(
+            "2 + 3 * 4 / 2" to 10,
+            "2 * 3 / 4 - 2" to -1,
+            "2 / 3 + 4 * 2" to 8
+        ).forEach { (expression, expected) ->
+            "$expression = $expected" {
+                stringCalculator.calculate(expression) shouldBe expected
+            }
+        }
+    }
+
+    "입력값이 널이거나 비어있을 경우" - {
+        listOf(
+            null,
+            " ",
+        ).forEach {
+            "expression: [${it ?: "null"}]" {
```

Comment:
```suggestion
            "expression: [${it.toString()}]" {
```
같은 결과가 나옵니다. 요 함수는 꼭 내부를 확인해보세요!

Reply:
감사합니다.